README.md

```diff
@@ -86,11 +87,13 @@ environment variable
 
 ##### Steps for Windows platform
 
-* Go to directory with cloned repository
+* Go to directory with cloned repository and enter subfolder tools\ms
```

Comment:
@orhan-mehmedov "Go to the directory with the cloned repository and enter the subfolder tools"